modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -754,6 +754,91 @@ func (suite *KeeperTestSuite) TestChanUpgradeOpen() {
 	}
 }
 
+// TestChanUpgradeOpenCounterPartyStates tests the handshake in the cases where
+// the counterparty is in a state other than OPEN.
+func (suite *KeeperTestSuite) TestChanUpgradeOpenCounterPartyStates() {
```

Comment:
think we normally refer to `counterparty` as one word

```suggestion
func (suite *KeeperTestSuite) TestChanUpgradeOpenCounterpartyStates() {
```